scripts/deploy/contracts/InvestorVesting.ts

```diff
@@ -6,7 +6,7 @@ import hre, { ethers } from "hardhat";
 export const contractNames = () => ["InvestorVesting"];
 
 const tokenAddress = "0xd9BAcC5BccAd9A380001d41Cd234b4D5f33ece76";
-const startDate = 1660201727;
+const startDate = 1660274000;
```

Comment:
lets use new Date() + few days, so unit test will always working.

Reply:
I handle that in the unit test file itself. I think it is nice to have the time hardcoded so you can't accidentally deploy a contact with the incorrect time (as if the starttime is in the past the constructor will throw)